.circleci/config.yml

```
@@ -168,11 +166,15 @@ step_run_all_tests: &step_run_all_tests
   run:
     name: Run tests
     command: |
+        # Ensures it's possible to debug hung tests in CI
+        echo "--format=documentation" > .rspec-local
```

Minor: Maybe use `>>` here as well, so that it doesn't accidentally clobber other configurations in the future?

```suggestion
        # Ensures it's possible to debug hung tests in CI
        echo "--format=documentation" >> .rspec-local
```

---

I left it as `>` on purpose, to ensure it was a clean file, but I see the opposite use case as well.

---

That's a reasonable point.

I guess we could add a failure if the file exists (e.g. something like `cat .rspec-local && echo "File not expected to exist" && exit 1`) and then we'd get a best of both worlds, although in this tiny case it may not be worth the trouble :)